modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/TopLevelBuild.java

```diff
@@ -1332,6 +1348,7 @@ protected boolean isCompareToUpstream() {
 
 	private boolean _compareToUpstream = true;
 	private long _lastDownstreamBuildsListingTimestamp = -1L;
+	private PullRequest _pullRequestObject;
```

Comment:
You should rename this attribute to just `_pullRequest`. The word, "Object" doesn't add anything